scripts/dev_setup.sh

```diff
@@ -589,6 +594,37 @@ function install_deno {
   chmod +x "${INSTALL_DIR}deno"
 }
 
+function install_swift {
+    echo Installing Swift.
+    #if [[ "$PACKAGE_MANAGER" == "apt-get" ]]; then
+    apt-get update
+    echo "AVAIL PKGS: $(apt-cache search libstdc++)"
+    echo "GLIBC VERSION: $(dpkg -l libc-bin)"
+    install_pkg wget "$PACKAGE_MANAGER"
+    install_pkg apt-get install  "$PACKAGE_MANAGER"
```

Comment:
This line looks funky?

Reply:
Yea, was an accidental include in the midst of debugging. Removed now :)